adapters/taboola/taboola.go

```diff
@@ -35,9 +38,15 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		return nil, []error{err}
 	}
 
+	parsedUrl, err := url.Parse(a.externalUrl)
```

Comment:
@ahmadlob Thanks for addressing my comments. Can you please move this URL parsing logic to the `Builder` function? The `externalUrl` value doesn't change with each request so we can only parse it once at start up and re-use it on a per request basis

Reply:
@mansinahar  Thanks for the suggestion, I agree it's a best practice for this logic to be in the `Builder`.
It's done :)